.github/workflows/docker-image-build.yml

```diff
@@ -0,0 +1,153 @@
+name: Build Images
+
+on:
+  push:
+    branches:
+    - master
+    - multi-arch
+    tags:
+      - '*'
+
+jobs:
+  BuildBase:
+    runs-on: ubuntu-latest
```

Comment:
Maybe we should append a specified version for ubuntu instead of latest.

Reply:
Good point.